rakelib/yard_builder.rb

```diff
@@ -59,6 +67,21 @@ def rebuild_all
 
   protected
 
+  def case_insensitive_fs?
+    File.exists? master_dir + "RAKEFILE"
```

Comment:
This is checking the main repo, not the repos that get created in tmp. I think we can improve this check.